examples/hyperparameter/hyperparameter.py

```diff
@@ -0,0 +1,80 @@
+from typing import List
+
+import orchpy
+import time
+import numpy as np
+
+@orchpy.distributed([tuple], [np.ndarray])
```

Comment:
why take a tuple of (x, y) instead of two arguments? e.g., 
```python
@ray.remote([np.ndarray, np.ndarray], [np.ndarray])
def linreg_gd(x, y):
  ...
```